packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -222,9 +124,17 @@ Example:
 </head>
 ```
 
-## Next steps
+## FAQs
```

Comment:
I don't understand why you have these two questions here. The first answer should be in the note under "About Error Page Customization" and the second answer is already covered in your opening sentence under "Edit the Error Page"

Reply:
Good point. These questions were legacy content that I thought would work in a separate FAQs section. But your comments make me wonder if we need the section at all.